docs/conf.py

```diff
@@ -171,3 +173,26 @@
 towncrier_draft_include_empty = False
 
 extensions += ['jaraco.tidelift']
+
+# Add icons (aka "favicons") to documentation
+sys.path.append(os.path.join(os.path.dirname(__file__), '_ext'))
+extensions += ['_custom_icons']
+
+# List of dicts with <link> HTML attributes
+# as defined in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
+# except that ``file`` gets replaced with the correct ``href``
+icons = [
+    {  # "Catch-all" goes first, otherwise some browsers will overwrite
+        "rel": "icon",
+        "type": "image/svg+xml",
+        "file": "images/logo-symbol-only.svg",
+        "sizes": "any"
+    },
+    {  # Version with thicker strokes for better visibility at smaller sizes
+        "rel": "icon",
+        "type": "image/svg+xml",
+        "file": "images/favicon.svg",
```

Comment:
Maybe call the file properly instead of having a comment only in one place?

Reply:
Hi @webknjaz, I am not sure if I understood correctly your comment.
Do you mean having a `"relative_path"` instead of `"file"` in the dictionary? That would make sense...

Reply:
No, I'm talking about the filename. It's not cool that the file is called `favicon.svg` and needs a code comment above explaining what it is. This information should be present in a file name, treat it as a variable in code. If the name is self-explanatory, then it'd be useful when checking out the HTML output, or just opening the URL in a browser, or while browsing the repository. A name like `favicon` does not give any hint about what the difference is from the other SVG file, nor it points at the relation between this and `logo-symbol-only.svg`.
You've added two code comments for each entry. The first one is quite useful because it explains the motivation but this second one has both useful and pointless bits. If a code comment has to include explanations about what the things are, you're probably calling those things wrong names.